CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/MergeNodesPlugin.java

```diff
@@ -40,8 +40,8 @@
 import au.gov.asd.tac.constellation.plugins.parameters.types.SingleChoiceParameterType;
 import au.gov.asd.tac.constellation.plugins.parameters.types.SingleChoiceParameterType.SingleChoiceParameterValue;
 import au.gov.asd.tac.constellation.plugins.templates.SimpleQueryPlugin;
-import au.gov.asd.tac.constellation.views.dataaccess.DataAccessPlugin;
-import au.gov.asd.tac.constellation.views.dataaccess.DataAccessPluginCoreType;
+import au.gov.asd.tac.constellation.views.dataaccess.plugins.DataAccessPlugin;
+import au.gov.asd.tac.constellation.views.dataaccess.plugins.DataAccessPluginCoreType;
```

Comment:
@formalhaunt can you please update the `CHANGELOG.md` with a reference to this change because it will require an update to Data Access Plugins in other repositories like https://github.com/constellation-app/constellation-adaptors and https://github.com/constellation-app/constellation-training. If you are OK to PR those as well then that would be great but I am happy to help and do those otherwise.

Reply:
Added the change to the `CHANGELOG.md`. Happy to do the PR changes for the other projects as well.